internal/capability/operand_cleanup.go

```diff
@@ -11,17 +11,7 @@ import (
 )
 
 // OperandCleanup removes the operand from the OCP cluster in the ca.namespace
-func operandCleanup(ctx context.Context, opts ...auditOption) auditCleanupFn {
-	var options auditOptions
```

Comment:
Why is this being removed?

Reply:
I mean, I see what you are doing, but is it actually a problem? It seems to me that we would want to keep the functional options separated, and not rely on the processing of one to the other.

Reply:
It's sharing the options from operator install.  @yashoza19 and I spent some time checking on those. New options for cleanup don't have the alm examples to delete the operands. In the end of the day they wouldn't be deleted anyway. Even updating the unstructured object with no finalizers.

Reply:
Yes. It's a problem because we need the same data the operand install uses at least on the extra custom resources field (aka ALM Examples)

Reply:
If you think this is the best way to fix it, go for it.